packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -205,11 +224,20 @@ export const ChatList = ({
     onClick: () => markAllThreadsAsRead(chatListState?.chatThreads)
   };
 
+  const isLoading = [
+    'creating server connections',
+    'server connection established',
+    'subscribing to notifications',
+    'loading messages'
+  ].includes(chatListState?.status ?? '');
+
+  const isError = ['server connection lost', 'error'].includes(chatListState?.status ?? '');
+
   return (
     // This is a temporary approach to render the chatlist items. This should be replaced.
```

Comment:
i know this isn't part of the work but I suggest to remove this comment since we are ready